### DISCUSSION TITLE
Laravel freezes locally

### DISCUSSION THREAD

**Discussion by vsotreshko** [qruto/laravel-wave#28] [Q&A]
Hi,

I am trying to test `laravel-wave` but application freezes when I am opening one page with SSE locally. I mean that when I open the second tab with another page, it is not loading.

I will be really thankful for any help

**Comment by slavarazum** [comment]
Which environment do you use?
Try to adjust max connections for your web server.

**Reply by vsotreshko** [reply]
Hi, sorry for late answer, I didn't receive notification.

I am trying to setup locally (on localhost). It is standart Laravel setup with sail.

**Reply by slavarazum** [reply]
Hi. Try to update to latest package version.

**Reply by DrAnax** [reply]
If your application freezes, it means you did not start the worker job. Try `php artisan queue:work`. This should fix things up.

**Reply by slavarazum** [reply]
It's better to use `sync` queue driver for debugging.

**Comment by slavarazum** [comment]
If you still have obstacles, feel free to open an issue providing more details about your case.
